code/_compile_options.dm

```diff
@@ -70,4 +70,8 @@
 #endif
 
  #define SERVICE_CMD_PARAM_KEY "serviceCommsKey"
- 
\ No newline at end of file
+
+//Update this whenever the db schema changes 

```

Comment:
@MrStonedOne I assume you need to update this file manually since it contains the service comms key and other shiz?